packages/react/src/BacktraceClient.ts

```diff
@@ -43,21 +41,24 @@ export class BacktraceClient extends BrowserClient {
      * @param build builder
      * @returns backtrace client
      */
-    public static initialize(options: BacktraceConfiguration, build?: (builder: BacktraceReactClientBuilder) => void) {
-        if (this._instance) {
-            return this._instance;
+    public static initialize(
+        options: BacktraceConfiguration,
+        build?: (builder: BacktraceReactClientBuilder) => void,
+    ): BacktraceClient {
+        if (this.instance) {
+            return this.instance;
         }
         const builder = this.builder(options);
         build && build(builder);
         this._instance = builder.build();
-        return this._instance;
+        return this._instance as BacktraceClient;
     }
 
     /**
      * Returns created BacktraceClient instance if the instance exists.
      * Otherwise undefined.
      */
     public static get instance(): BacktraceClient | undefined {
-        return this._instance;
+        return this._instance as BacktraceClient;
     }
```

Comment:
You should be able to move this to `BacktraceCoreClient` and mark it to return `this | undefined`.

Reply:
goood idea.

Reply:
I checked your idea and there is a problem with the instance variable type - we use it to make sure client is a singleton type. In this situation, we cannot type it to `this` 

![image](https://github.com/backtrace-labs/backtrace-javascript/assets/9386268/efa0aa5a-b1f9-4533-8efc-b0f615a176ae)